Content.Server/TimerStopwatch/TimerStopwatchSystem.cs

```diff
@@ -0,0 +1,68 @@
+using Content.Server.Popups;
+using Content.Shared.Interaction;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Player;
+using System;
+
+namespace Content.Server.TimerStopwatch
+{
+    //NOTE: If named "stopwatch" causes conflicts with the engine stopwatch system
+    public class StopwatchSystem : EntitySystem
+    {
+        [Dependency] private readonly PopupSystem _popupSystem = default!;
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<TimerStopwatchComponent, UseInHandEvent>(OnUseInHand);
+            SubscribeLocalEvent<TimerStopwatchComponent, GetInteractionVerbsEvent>(AddInteractionVerbsVerbs);
+        }
+
+        private void AddInteractionVerbsVerbs(EntityUid user, TimerStopwatchComponent component, GetInteractionVerbsEvent args)
+        {
+            //TODO: Add verb icons
+            Verb trackverb = new();
+            trackverb.Act = () => component.trackingStatus = !component.trackingStatus;
+            trackverb.Text = Loc.GetString("timekeeping-verb-toggle");
+            // trackverb.IconTexture = "/Textures/Interface/VerbIcons/eject.svg.192dpi.png";
+            Verb resetverb = new();
+            resetverb.Act = () => component.passedTime = 0;
+            resetverb.Text = Loc.GetString("timekeeping-verb-reset");
+            // resetverb.IconTexture = "/Textures/Interface/VerbIcons/eject.svg.192dpi.png";
+            args.Verbs.Add(trackverb);
+            args.Verbs.Add(resetverb);
+        }
+
+        /// <summary>
+        /// Interaction with the clock, tells the time.
+        /// </summary>
+        private void OnUseInHand(EntityUid uid, TimerStopwatchComponent comp, UseInHandEvent args)
+        {
+            TellTime(comp.passedTime, args.User);
+        }
+
+        /// <summary>
+        /// Tells the time of a given timekeeping device
+        /// </summary>
+        public void TellTime(float rawtime, EntityUid user)
+        {
+            var timeSpan = TimeSpan.FromSeconds(rawtime);
+            _popupSystem.PopupEntity(timeSpan.Minutes.ToString() + ":" + timeSpan.Seconds.ToString(), user, Filter.Entities(user));
+        }
```

Comment:
If IItemStatus has something updated per frame seems easier to just use that?

Reply:
What do you mean?

Reply:
There's an ItemStatus UI thing that shows up above hands if you want to implement that. Not sure if that's what @metalgearsloth is talking about, but might be.

Reply:
Yeah it seems more reasonable to just show the time, rather than generating a popup if a user used the watch in their hands.

For more info on the UI: its the same things tasers & flashlights use for displaying the charge remaining, so I guess poke around in those files too see how they use `IItemStatus`